c_glib/doc/arrow-glib/arrow-glib-docs.xml

```diff
@@ -163,6 +163,10 @@
     <title>Index of deprecated API</title>
     <xi:include href="xml/api-index-deprecated.xml"><xi:fallback /></xi:include>
   </index>
+  <index id="api-index-0-13-0" role="0.13.0">
+    <title>Index of new symbols in 0.13.0</title>
+    <xi:include href="xml/api-index-0.12.0.xml"><xi:fallback /></xi:include>
```

Comment:
Could you replace the "0.12.0" with "0.13.0"?

Reply:
Thank you for pointing out it. I've fixed it.